rosdep/base.yaml

```diff
@@ -3408,6 +3408,17 @@ libvtk-qt:
     xenial: [libvtk6-qt-dev]
     yakkety: [libvtk6-qt-dev]
     zesty: [libvtk6-qt-dev]
+libvulkan-dev:
+  debian:
+    buster: [libvulkan-dev]
+    stretch: [libvulkan-dev]
+  fedora: [vulkan-devel]
+  gentoo: [media-libs/vulkan-loader]
+  ubuntu:
```

Comment:
I think that we require rosdep package availability on all active ubuntu distributions.
This package is not available on ubuntu trusty.
It should either be backported to trusty and imported on the farm or be released as a ROS package in ROS Indigo. @tfoote @wjwwood @dirk-thomas to confirm

Reply:
Well, as long as the key is enumerated like this, I don't see why we wouldn't allow it. If you try to use this key in a package being released into Indigo it will fail still (as before). I'll let @dirk-thomas or @tfoote reply.

Reply:
That was my thinking in exhaustively listing the ubuntu distributions; trusty users will see the same behaviour they saw before this change.

Reply:
The opposite approach would just not require any future maintenance:

```
ubuntu:
  '*': [libvulkan-dev]
  trusty: null
```

I don't mind either or.

Reply:
Changed to the wildcard/exclusion approach in ef1113a.

Hoping this can be squashed before merging to avoid polluting your history.

Reply:
Thanks for the updates.

> Hoping this can be squashed before merging to avoid polluting your history.

Absolutely.